templates/es-cluster/7/rancher-compose.yml

```diff
@@ -99,6 +99,15 @@ catalog:
             - rancher-efs
             - rancher-ebs
 
+        - variable: "VOLUME_DRIVER_OPTS"
+          description: "VOLUME driver option"
+          description: |
+            Specify a single "driver_opts" key/value pair in the format "optionName: optionValue".
+            E.g. for the `rancher-ebs` driver you should specify the required 'size' option like this: "size: 1".
+          label: "Volume Driver Option (Optional)"
+          required: false
+          type: string
```

Comment:
What do you think to use type `multiline` instead of `string`?? One driver option by line could be added..
```
-          type: string
+          type: multiline
```